app/services/s3_query_service.rb

```diff
@@ -119,6 +119,8 @@ def find_s3_file(filename:)
   # Retrieve the S3 resources uploaded to the S3 Bucket
   # @return [Array<S3File>]
   def client_s3_files(reload: false, bucket_name: self.bucket_name, prefix: self.prefix)
+    return [] if Rails.env.development?
```

Comment:
@jrgriffiniii this is causing me issue in development.  I just have a script locally that exports the staging environment variables so I can connect to AWS in development.  This causes things not to work for me on this branch.